source/proof_address_mock/lib/proof_address_mock.rb

```diff
@@ -30,7 +30,9 @@ def proof(&callback_block)
       result[:context] = { stages: [
         address: IdentityIdpFunctions::AddressMockClient.vendor_name
       ] }
+
       result[:timed_out] = proofer_result.timed_out?
+      result[:exception] = proofer_result.exception.inspect if proofer_result.exception
```

Comment:
previously we were only returning the message of the exception, but calling `inspect` means we get `"#<RuntimeError: there was an error>"` instead of `"there was an error"`